dvc/repo/__init__.py

```diff
@@ -129,19 +129,24 @@ def __repr__(self):
 
     @classmethod
     def find_root(cls, root=None):
-        if root is None:
-            root = os.getcwd()
-        else:
-            root = os.path.abspath(os.path.realpath(root))
+        root_dir = os.path.abspath(os.path.realpath(root or os.getcwd()))
```

Comment:
To avoid cwd -> relpath -> abspath in case of root == None:
we could `root_dir= os.path.abspath(os.path.relpath(root)) if root else os.getcwd()`

Reply:
Just like what was before? I'll just rollback.

Reply:
as @skshetry noted, abspath is not necessary at all here